packages/messaging/src/Message.utils.ts

```diff
@@ -244,19 +239,19 @@ export function errorCheckMessage(message: IMessage): void {
  *
  * @param requiredProperties The list of required properties that need to be verified against a [[CType]].
  * @param cType A [[CType]] used to verify the properties.
- * @throws [[ERROR_CTYPE_HASH_NOT_PROVIDED]] when the properties do not match the provide [[CType]].
+ * @throws [[CTypeHashMissingError]] when the properties do not match the provide [[CType]].
```

Comment:
This seems to rather throw `CTypeUnknownPropertiesError`, plus the ones from `CType.verifyDataStructure`. Is it best practice to also mention all the possible errors of sub-functions?

Reply:
Frankly, this is the first project I see utilizing `@throws` in the JSDoc, so I have no idea. For the consumer of the JSDoc it doesn’t make much sense to only see what the function itself throws. On the other hand, without language-level support, it is unmaintainable to list the exceptions thrown from sub-functions. However since this function doesn’t throw anything by itself, and still has `@throws` in JSDoc, I assumed that you do list exceptions of sub-functions. I could just remove this line completely.